### DISCUSSION TITLE
Market Research_NFT Tarot

### DISCUSSION THREAD

**Discussion by evolayak** [SecretDecks/Documentation#19] [Show and tell]
Hi all,
Was doing some research on existing NFT Tarot decks and anything Tarot-NFT related, so adding here. I’ll periodically update the list, this isn’t exhaustive; please add as you see fit.  (Hope this is the right place for this discussion hehe) 

1. [BitcoinTarot - World's 1st Crypto Tarot Cards](https://opensea.io/collection/cryptotarot) 2019; [“Predicting the Crypto Markets with the World's 1st Crypto-themed Tarot Cards”](https://www.issuewire.com/predicting-the-crypto-markets-with-the-worlds-1st-crypto-themed-tarot-cards-1638201518337431) OpenSea

2. [The Fool_Aaron Gombar](https://www.blockchainartexchange.co.uk/product/the-fool/) 2020, Blockchain art exchange 

3. [Twisted Vacancy Major Arcana (part 1)](https://niftygateway.com/collections/twistedtarot) Nifty Gateway

4. [Custom Tarot Layout from](http://tarot.finance/product/tarot-card-layout-nft/) Tarot.finance - just a reading; also [Tarot Pools Coming Soon](http://tarot.finance/). Site and platform look sketch 

5. [New Product Launch: Trader Tarot Cards](https://www.tradingview.com/blog/en/new-product-launch-trader-tarot-cards-23658/), which was just an [April Fools joke from TradingView](https://www.tradingview.com/chart/TSLA/63LZK106-April-Fools/).

6. [Bitsozza collab project_Trionfi 3D deck](https://foundation.app/bitsozza) April 2021

7. [Deep Dream - Major Arcana](https://opensea.io/collection/tarot-de-marsella-deep-deam-major-arcana) April 2021, OpenSea

8. [CRYPTO TAROT CARDS collab_Ann Pokes x Bleykhera](https://rarible.com/cryptotarotcards) April 2021, Rarible

9. [WOCA Arcana Crypto Tarot](https://womenofcrypto.art/tarot/) April 2021, Aysync

[interesting FT article on the Tarot's rising popularity](https://www.ft.com/content/c4afbc05-a715-4b83-9323-44e4c4f95ca5)

**Comment by jonloone** [comment]
This might be slightly off topic, but I'm curious about existing tarot-based apps on iOS / Android and how we might partner with them for onboarding artists they have or borrow some concepts for readings.

These  come to mind

https://labyrinthos.co/
https://goldenthreadtarot.com/
https://www.biddytarot.com/

**Comment by evolayak** [comment]
Cool, thanks for this and nice idea! I refer to biddytarot sometimes and some of her interpretations really resonate with me.

**Comment by Danzig420** [comment]
I`m a NFT Occult Artist who has been working on implementing the occult into the nft. I have a set of runes, and a zodiac set almost done. I`ve started, on my tarot. I came here, because I was doing research on making a functional nft set. I also was looking for like minds for other such occult projects, and expansion. here is my opensea  https://opensea.io/Kaos_Kreations . I would love to see where this goes.

**Reply by DanM3rcurius** [reply]
Hello and wow!
First wow that you came across our repo and second wow that you pointed me to this discussion I forgot about ^^
Third wow for the powerful imagery you have created there!
We're currently focusing on offering tools for self study, you can check out our Arcana GitBook https://secretdecks.gitbook.io/arcana/
or join the discord if you want to get started on collabs or want to more about the DisCo side of SecretDecks 🙏🏿
